helpdesk/views/staff.py

```diff
@@ -471,30 +471,19 @@ def subscribe_to_ticket_updates(ticket, user=None, email=None, can_view=True, ca
                 _('When you add somebody on Cc, you must provide either a User or a valid email. Email: %s' % email)
             )
 
-        ticketcc = TicketCC(
-            ticket=ticket,
+        return ticket.ticketcc_set.create(
             user=user,
             email=email,
             can_view=can_view,
             can_update=can_update
         )
-        ticketcc.save()
-
-        return ticketcc
-
-
-def subscribe_staff_member_to_ticket(ticket, user, email='', can_view=True, can_update=False):
-    """used in view_ticket() and update_ticket()"""
-    return subscribe_to_ticket_updates(ticket=ticket, user=user, email=email, can_view=can_view, can_update=can_update)
 
```

Comment:
Nice rationalisation of the code.

Reply:
Good catch! There might be a need for some staff specific logic but we can always add that back later when needed since this function doesn't seem to have done anything.